dora/tests/src/test/java/alluxio/client/cli/fsadmin/command/DoctorCommandIntegrationTest.java

```diff
@@ -15,11 +15,13 @@
 import alluxio.client.cli.fsadmin.AbstractFsAdminShellTest;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Tests for doctor command.
  */
+@Ignore
```

Comment:
i think its fine to remove this integration test if it is going to be completely ignored

Reply:
let me discuss with @jiacheliu3  to see if this needs to be removed as part of master cleanup exercise, for now i
ll just keep entire thing ignored.